### ISSUE TITLE
Is there a forecast for Grid view fix?

### ISSUE THREAD

**Issue by kaligem** [Fugiman/google-meet-grid-view#239]
_Before opening an issue, please check the existing open issues to see if somebody has already reported it! If something is horribly wrong, like all videos showing up in a single column, chances are somebody already reported it. Please reply in that issue instead._

**Describe what is broken**
A clear and concise description of what isn't working.

**Expected behavior**

**Screenshots**

**Software used:**
 - OS [e.g. Windows 10]:
 - Browser [e.g. Chrome, Firefox]:
 - Version [e.g. 22]:
 - Extension [e.g. Official Chrome Extension, Tampermonkey]:
 - Grid View Version [e.g. 1.19, 1.23]:

**Additional context**

**Comment by Eagle-Man** [comment]
No, there isn't, as it depends on the complexity of the problem, the amount that needs to be changed to fix it, and the amount of time that Fugiman has to work on it (only in his free time).  Submitting a ticket for something like this only serves to distract him from his work on the project.  he will certainly submit a new version once it's ready.  In the mean time, subscribe to Issue #225 to keep informed on this issue.  He will also likely post there when there is news.
